pandas/_libs/tslib.pyx

```diff
@@ -54,7 +52,8 @@ from tslibs.timestamps cimport (create_timestamp_from_ts,
                                 _NS_UPPER_BOUND, _NS_LOWER_BOUND)
 from tslibs.timestamps import Timestamp
 
-cdef bint PY2 = str == bytes
+
+DEF PY2 = str == bytes
```

Comment:
we should define in util.pxd maybe (future ok)

Reply:
I doubt that works for DEF values, but not much downside.